src/Symfony/Component/Security/Http/composer.json

```diff
@@ -27,6 +27,7 @@
     "require-dev": {
         "symfony/cache": "^5.4|^6.0",
         "symfony/expression-language": "^5.4|^6.0",
+        "symfony/http-client": "^6.0",
```

Comment:
We must add HttpClient package as suggested packages as well

Reply:
That's done thanks ! :-)